Source/Particles/ElementaryProcess/QEDSchwingerProcess.H

```diff
@@ -28,6 +28,8 @@ struct SchwingerFilterFunc
      * @param[in] i index of the cell in the first direction.
      * @param[in] j index of the cell in the second direction.
      * @param[in] k index of the cell in the third direction.
+     * @param[in] engine the random number state and factory
+     * @return ...?
```

Comment:
@lucafedeli88 or @NeilZaim maybe? :)
```suggestion
     * @return ...?
```

Reply:
@lucafedeli88 or @NeilZaim maybe? :)

Reply:
```suggestion
     * @return the number of pairs created in the cell.
```